queue_services/entity-filer/requirements.txt

```diff
@@ -26,4 +26,4 @@ reportlab==3.6.12
 git+https://github.com/bcgov/sbc-connect-common.git#egg=gcp-queue&subdirectory=python/gcp-queue
 git+https://github.com/bcgov/business-schemas.git@2.18.27#egg=registry_schemas
 git+https://github.com/bcgov/lear.git#egg=entity_queue_common&subdirectory=queue_services/common
-git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
+# git+https://github.com/bcgov/lear.git#egg=legal_api&subdirectory=legal-api
```

Comment:
If you want to play around this experimental code, you need to manually install legal-api and sql-versioning by `pip install -e <path>` (or build the wheels and install it)